contrib_bots/lib/john.py

```diff
@@ -0,0 +1,118 @@
+# pip install chatterbot is necessary
+import json
+import os
+import time
+
+from random import choice
+
+from chatterbot import ChatBot
+from chatterbot.trainers import ChatterBotCorpusTrainer, ListTrainer
+
+JOKES_PATH = os.path.expanduser('lib/John/jokes.json')
+DATABASE_PATH = os.path.expanduser('~/JohnBot/database.db')
```

Comment:
I personally don't like it when an application creates a non-hidden directory on my home. Could this be better if it were a hidden path?